pkgs/by-name/qs/qsv/package.nix

```diff
@@ -21,17 +21,21 @@ rustPlatform.buildRustPackage {
     owner = "jqnatividad";
     repo = "qsv";
     rev = version;
-    hash = "sha256-erXMDZBkOPnZdhhjnUTiS0eCuFo1v5sNhLn8o7QN/5g=";
+    hash = "sha256-27oSk8fnTvl1zJ8xYkZHkWVTq+AVDn4Zi1s56T49T1Q=";
   };
 
   cargoLock = {
     lockFile = ./Cargo.lock;
     outputHashes = {
-      "calamine-0.25.0" = "sha256-LFQahQ+SfGWhp5Kfs7AJCj4zyxWiz099pwVuQucCF00=";
+      "calamine-0.26.1" = "sha256-3WLXR6ZQoAi6/HtfEouCg9mR5E8iAt8Qzy3fZIrZbKA=";
+      "csv-1.3.0" = "sha256-QP+N76BTvoHyWOezl+ICwOOqEIfoJBt6PBouxZ6EZks=";
+      "csvlens-0.10.1" = "sha256-w1nRXz1g4MHKbp3GQ9osZGi4bTDzTdQjfKevl1/4DHA=";
       "dynfmt-0.1.5" = "sha256-/SrNOOQJW3XFZOPL/mzdOBppVCaJNNyGBuJumQGx6sA=";
-      "grex-1.4.5" = "sha256-4Tr5L87HuiUW8tJdqr4oT1yQXviU7XtDrKY7iYNcwbo=";
+      "grex-1.4.5" = "sha256-g8yPcnDV7hu/K8Ib8j8n+Qz9O7G4zdbMnQopBBn4mr0=";
       "local-encoding-0.2.0" = "sha256-ThXYKr3u/n2kvINcyobB2Ayex2sNbJEOyyjZH993Z4U=";
-      "polars-0.41.3" = "sha256-8xkcJgfKo7BzeBnR6XVdbcH9ZY9Kh4dcGFMmyA5LuQg=";
+      "polars-0.44.2" = "sha256-MqQoHLluWWkWFOK4+Njq/NnYm30XGze9S9cJ7ysPLOw=";
+      "publicsuffix-2.2.3" = "sha256-CV7ey5v1Lma+7q7dL4UfwSy7GjtK/OZvmrbyexFOfBY=";
+      "strum-0.26.3" = "sha256-R3bPqTwxyts7uYSas/3Ij39/m64DH0A++DMIdYqmSqA=";
     };
   };
```

Comment:
```nix
  useFetchCargoVendor = true;
  cargoHash = "sha256-tu9HCFAxmmYVgmJyHunBtGSqKGzwbX2vi6ju4cv33wc=";
```

Using this for me locally worked like a charm. Enabled by https://github.com/NixOS/nixpkgs/pull/349360.

Reply:
I forgot it existed, will change to use the library function